Bindings/Java/Matlab/Dynamic_Walker_Example/Main_WalkerForwardSim.m

```diff
@@ -0,0 +1,68 @@
+% ----------------------------------------------------------------------- 
+% The OpenSim API is a toolkit for musculoskeletal modeling and           
+% simulation. See http://opensim.stanford.edu and the NOTICE file         
+% for more information. OpenSim is developed at Stanford University       
+% and supported by the US National Institutes of Health (U54 GM072970,    
+% R24 HD065690) and by DARPA through the Warrior Web program.             
+%                                                                         
+% Copyright (c) 2005-2013 Stanford University and the Authors             
+% Author(s): Daniel A. Jacobs                                             
+%                                                                         
+% Licensed under the Apache License, Version 2.0 (the "License");         
+% you may not use this file except in compliance with the License.        
+% You may obtain a copy of the License at                                 
+% http://www.apache.org/licenses/LICENSE-2.0.                             
+%                                                                         
+% Unless required by applicable law or agreed to in writing, software     
+% distributed under the License is distributed on an "AS IS" BASIS,       
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         
+% implied. See the License for the specific language governing            
+% permissions and limitations under the License.                          
+% -----------------------------------------------------------------------
+
+% Reset
+clear all; close all; clc
+
+% Import
+import org.opensim.modeling.*;
+
+% Open a Model by name
+osimModel = Model('../Model/WalkerModel.osim');
+
+% Use the visualizer (must be done before the call to init system)
+osimModel.setUseVisualizer(true);
+
+% Initialize the system and get the initial state
+osimState = osimModel.initSystem();
+
+% Set the initial states of the model
+editableCoordSet = osimModel.updCoordinateSet();
```

Comment:
`setValue()` is a const method, so AFAIK you don't need the CoordinateSet to be editable (i.e., you can use `getCoordinateSet()` and remove "editable").